### ISSUE TITLE
Reverb and RTF 

### ISSUE THREAD

**Issue by JBloodless** [Brainchip-Inc/aTENNuate#1]
Hi, thanks for the great work and contribution. 
Two questions:
1) Did you test dereverberation quality (or at least did you use RIRs at training)?
2) What is RTF of this model? Latency values in the paper are quite vague, RTF would be much more helpful stat?

Thanks in advance

**Comment by PeaBrane** [comment]
Hi, thank you for your questions.

1. We did try to apply the model directly on the dns1 testset reverb, and it works quite well, even though not directly including RIRs durinf training. Some of the real recordings are also quite "reverbish", and the model seemed to handle it OK. We are looking into adding RIRs into training as well soonish.

2. We did not report the RTF (real-time factor) in the paper as it is hardware dependent, and thought the MACs / sec measure served better as a throughput proxy. I believe the RTF is around 20 when running on my laptop cpu, which is a macbook air M1. This is using a semi-optimized inference pipeline not included in the repo.
